src/hooks/useKeyboardControls.ts

```diff
@@ -63,7 +66,7 @@ export const useKeyboardControls: ReactHook<UseKeyboardControlsProps, void> = ({
     if (row - 1 < 1 || isSeatTaken(row - 1, column)) return;
 
     takeSeat(row - 1, column);
-    onMove && onMove();
+    onMove && onMove(); // TODO: do we need this here anymore?
```

Comment:
- [ ] [TODO] do we need this here anymore? (probably not)

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/0fcc08e4ad8d690b9dcd1d9fdeef07ca79f88caa